aegis_control/README.md

```diff
@@ -0,0 +1,17 @@
+# aegis_control
+
+This package contains the [ros2_control](https://control.ros.org/) launch files of the Aegis robot station, which uses configuration files from the `aegis_description` package.
+
+## Launch files
+
+```
+aegis_control/
+├── launch
+│   ├── start_drivers.launch.py
+│   └── ur_driver.launch.py
+```
+
+| File                                                        | Description                                                                                                                           |
+| ----------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| [start_drivers.launch.py](./launch/start_drivers.launch.py) | The main launch file to run whole Aegis' ros2_control stack.                                                                          |
+| [ur_driver.launch.py](./launch/ur_driver.launch.py)         | Launches nodes from the [ur_robot_driver](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver) to control the UR5e robot. |
```

Comment:
Add development node regarding the `ur_robot_driver` 2.5.1, which requires their custom `control_manager` node for the real hardware usage.